### PR TITLE
lets go box collider on player

### PR DESCRIPTION
**Pull request by trilleplay**

### CODE REVIEW

File: Assets/Prefabs/Player.prefab [hunk 1]
```diff
@@ -278,20 +278,6 @@ MeshRenderer:
   m_SortingLayer: 0
   m_SortingOrder: 0
   m_AdditionalVertexStreams: {fileID: 0}
---- !u!136 &7006171859244352316
```
Comment (olchyk98) [inline Assets/Prefabs/Player.prefab:281]: MAY GAY

**Review by olchyk98 — CHANGES_REQUESTED** [review]

**Review by HugoLandin — APPROVED** [review]
lets go

**Comment by olchyk98** [comment]
RICHARD GJORDE EN TILL

**Comment by trilleplay** [comment]
https://www.youtube.com/watch?v=U2JyeciEwSI

**Pull request closed without merging**